packages/contracts/src/ubiquistick/UbiquiStickSale.sol

```diff
@@ -93,26 +92,28 @@ contract UbiquiStickSale is Ownable, ReentrancyGuard {
         // and had enough allowance with enough funds
         uint256 count;
         uint256 price;
+        uint256 paid = 0;
         (count, price) = allowance(msg.sender);
         require(
             count > 0,
             "Not Whitelisted For The Sale Or Insufficient Allowance"
         );
-
         if (remainingTokenCount < count) {
             count = remainingTokenCount;
+            paid = remainingTokenCount * price;
         }
         if (msg.value < count * price) {
+            paid = msg.value * price;
```

Comment:
Shouldn't it be `paid = count * price` inserted after this [line](https://github.com/ubiquity/ubiquity-dollar/pull/714/files#diff-6bb51bf2f1f9f376618af61a61a3adb284a5b42d097fc1c9e91973e0a3b0b810R107)?

Reply:
> Shouldn't it be `paid = count * price` inserted after this [line](https://github.com/ubiquity/ubiquity-dollar/pull/714/files#diff-6bb51bf2f1f9f376618af61a61a3adb284a5b42d097fc1c9e91973e0a3b0b810R107)?

sorry my bad i was supposed to remove that * price too its gonna be just msg.value